files/usr/share/cinnamon/cinnamon-settings/cinnamon-settings.py

```diff
@@ -422,7 +422,7 @@ def __init__(self):
             # can consider it as a standalone app and give it its own
             # group.
             wm_class = "cinnamon-settings %s" % arg1
-            self.window.set_wmclass(wm_class, wm_class)
+            self.window.set_role(wm_class)
```

Comment:
Removing a deprecation sounds useful, but why does this come with a commit message claiming it is relevant to python 3.8?